tests/jsx-significant-space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -101,24 +92,20 @@ before_break2 = (
     <span
       barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar
     />
-    {" "}
-    foo
+    {" "}foo
   </span>
 );
 
 after_break = (
   <span>
     foo
-    {" "}
-    <span barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar />
+    {" "}<span
```

Comment:
here it would be better to have the `{" "}` after `foo`, not before `<span>`. RIght?